lucene/analysis/common/src/java/org/apache/lucene/analysis/hunspell/Stemmer.java

```diff
@@ -588,6 +585,12 @@ private boolean checkCondition(
     return stems;
   }
 
+  private boolean isFlagAppendedByAffix(int affixId, char flag) {
+    if (affixId < 0) return false;
```

Comment:
Add curly braces. I know, it's verbose but some folks on this list even went as far as to require it (the formatter doesn't care).

Reply:
Curly braces and 2 more lines? Oh pity, I love conciseness so much :(

Reply:
The discussion of whether braces in single-line statements in if/while/for loops are needed or not
is like religion - you can't end it, once started... I'm just relaying what has been said 
on the mailing list after the formatter folded code like this:
```
if (foo)
  doSomething()
else
  doSomethingElse()
```

Privately, I don't have a strong opinion. Depends on the context where it's at, I guess.

Reply:
In that case I'd also put braces (although I've seen people who don't). But my case is "then-branch is a simple `return` on the same line". Are braces also required in this case? And separate lines, like this?
```
if (affixId < 0) {
  return;
}
```

Reply:
Required is not the right word... Perhaps... advised. :) You are entitled to your own opinion (at least as long as this passes precommit checks!) - like I said, I relayed what's been mentioned on the mailing list. Some people just put braces everywhere and they are used to this consistency, even if it adds verbosity.

Reply:
Would it be acceptable to reformat the code when such a demand appears, or after I'm finished with these improvements? These "some people" might never need to look into Hunspell code.

Reply:
I think consistency in those braces (over verbosity) is a good rule in general, but ok.